### ISSUE TITLE
Default factory name setting in build.gradle.kts not working for Android

### ISSUE THREAD

**Issue by JuliaJakubcova** [touchlab/compose-swift-bridge#5]
Hello,
I have multiple shared modules in my application and according to the [tutorial](https://touchlab.co/composeswiftbridge/tutorial) I wanted to set the default name for the factory:
```
ksp {
    // Configure the module with a custom default factory name, the default is called "NativeView"
    arg("compose-swift-bridge.defaultFactoryName", "SampleView")
}
```
It seems this setting only influences the iOS and common code generation, but the Android seems to ignore it and use `NativeView` name.
![Screenshot 2024-11-26 at 13 43 00](https://github.com/user-attachments/assets/6653b655-bea3-450d-bdb7-9d55cc585225)

I tried setting it in the `@ExpectSwiftView` and that worked as expected, Android generated factory had the correct name.

**Comment by DevSrSouza** [comment]
I was able to reproduce it! Thanks for reporting.

This seems to be a issue with KSP or Kotlin Compiler API, this code bellow returns `null` for iOS and returns `NativeView` for Android (that is the default parameter from the Annotation)

```kotlin
expectSwift
        ?.arguments
        ?.firstOrNull { it.name?.getShortName() == "factoryName" }
        ?.value as? String?
```

We will need to patch the generator.

**Comment by DevSrSouza** [comment]
We have publish a new version with this fix: `0.1.1`.
